### PR TITLE
Create mlbench-vs-mlperf.md

### PR DESCRIPTION
**Pull request by mmilenkoski**
This PR adds a draft blog post for the comparison between MLBench and MLPerf

### CODE REVIEW

**Comment by mmilenkoski** [comment]
@martinjaggi I merged the table as we discussed. Please let me know whether I can publish the post.

**Comment by ehoelzl** [comment]
@mmilenkoski Is the post ready to be published ?

**Review by ehoelzl — APPROVED** [review]
Seems great. Thank you !

File: _drafts/mlbench-vs-mlperf.md [hunk 1]
```diff
@@ -0,0 +1,149 @@
+---
+layout: post
+title: Comparison of MLBench and MLPerf
```
Comment (ehoelzl) [inline _drafts/mlbench-vs-mlperf.md:3]: If you could add yourself as author that would be nice. You need to add your entry in `data/authors.yml`

**Comment by mmilenkoski** [comment]
@ehoelzl I merged it, should I also move it to `_posts`? It is currently in `_drafts`

**Pull request merged**